lib/cprofiles/CMakeLists.txt

```diff
@@ -353,7 +353,6 @@ if(CPROF_SYSTEM_MACOS)
 
   if (CPACK_GENERATOR MATCHES "productbuild")
     set(CPACK_SET_DESTDIR "ON")
-    configure_file(cpack/macos/welcome.txt.cmakein ${CMAKE_CURRENT_SOURCE_DIR}/welcome.txt)
```

Comment:
this PR should not touch lib/cprofiles

Reply:
No but need to fix it to test macOS so merged that in for now